strawberry/contrib/django/converter.py

```diff
@@ -10,8 +10,8 @@
 @singledispatch
 def convert_django_field(field):
     raise ImproperlyConfigured(
-        "Don't know how to convert the Django form field %s (%s) "
-        "to type" % (field, field.__class__)
+        f"Don't know how to convert the Django model "
+        + "field {field} ({field.__class__}) to type"
```

Comment:
I'm not sure this will work as you want it to, you need `f` on the second line as well!

```
>>> def foo():
...     x = 1
...     print(f"foo" +
...           "{x}") # no f-string

>>> foo()
foo{x}

>>> def foo():
...     x = 1
...     print(f"foo" +
...           f"{x}") # f-string!

>>> foo()
foo1
```

Reply:
Good catch!